src/components/Comments/Comment.js

```diff
@@ -328,14 +328,14 @@ const styles = StyleSheet.create({
   username: {
     fontFamily: fonts.inter[700],
     fontSize: normalizeFontSize(12),
-    color: '#828282',
+    color: COLORS.blackgrey,
     lineHeight: 14,
     marginLeft: 16
   },
   post: {
     fontFamily: fonts.inter[400],
     fontSize: normalizeFontSize(14),
-    color: '#333333',
+    color: COLORS.mine_shaft,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hard-coded color values in the `username` and `post` styles have been replaced with dynamic colors from the theme object. This improves consistency and maintainability of the code.
```diff
-    color: '#828282',
+    color: COLORS.blackgrey,
...
-    color: '#333333',
+    color: COLORS.mine_shaft,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->